### PR TITLE
Add a few other maintainers

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -63,3 +63,6 @@ extra:
   recipe-maintainers:
     - isuruf
     - jakirkham
+    - kkraus14
+    - mike-wendt
+    - raydouglass
```
Comment (jakirkham) [inline recipe/meta.yaml]: @kkraus14 @mike-wendt @raydouglass, would you be ok being added as maintainers on the `nvcc` (shim compiler package) feedstock? Please let me know 🙂

Comment (kkraus14) [reply recipe/meta.yaml]: @jakirkham would be my pleasure! Suggest adding @dillon-cullinan as well.

Comment (mike-wendt) [reply recipe/meta.yaml]: @jakirkham same, happy to help and support @dillon-cullinan being added as well

File: recipe/meta.yaml [hunk 2]
```diff
@@ -61,5 +61,9 @@ about:
 
 extra:
   recipe-maintainers:
+    - dillon-cullinan
```
Comment (jakirkham) [inline recipe/meta.yaml]: @dillon-cullinan, I've added you as a maintainer here. Please let me know if that is ok. 🙂

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Thanks all! 😄 Have added Dillon as well. 👍

**Comment by jakirkham** [comment]
Going to go ahead and merge this. Hope that is ok 🙂

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Dillon, if you decide you don't want to be listed here, please just let us know and we can remove you. Thanks!

**Comment by jakirkham** [comment]
Please see [this comment]( https://github.com/conda-forge/nvcc-feedstock/commit/e5ac2f99025a94d7c75f94fb14ef8832c62db03f#commitcomment-35136266 ) for details about joining conda-forge. Welcome all! 😄

**Comment by jakirkham** [comment]
Forgot to re-render the feedstock to update other files following this change. Not a big deal, but it should make other changes cleaner and more compact. Handling this with PR ( https://github.com/conda-forge/nvcc-feedstock/pull/6 ).

**Pull request merged**